src/test/java/seedu/duke/parser/ParserTest.java

```diff
@@ -105,5 +126,37 @@ void parseAdd_wrongComponent_expectException() throws DukeException {
         assertTrue(c instanceof AddCommand);
     }
 
+    @Test
+    void parseCalc_valueEff_CalculateCommand() throws DukeException {
+        Parser p = new Parser();
+        p.parse("template rl");
+        String line = "calc ";
+        String[] valueEff = {"reff", "leff", "current", "power"};
```

Comment:
Same with this collection of values:
```suggestion
        String[] effValues = {"reff", "leff", "current", "power"};
```